Client/Application/AppDelegate.swift

```diff
@@ -157,10 +157,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
+        LeanplumIntegration.sharedInstance.setup(profile: profile)
+        LeanplumIntegration.sharedInstance.start()
+
         // We need to check if the app is a clean install to use for
         // preventing the What's New URL from appearing.
         if getProfile(application).prefs.intForKey(IntroViewControllerSeenProfileKey) == nil {
```

Comment:
Sorry @jhugman - isn't @bkmunar just doing what the Adjust initialization is doing right above?